components/studio/bin/hab-studio.ps1

```diff
@@ -455,6 +455,17 @@ function Set-Secrets {
   }
 }
 
+function Update-SslCertFile {
+  if($env:SSL_CERT_FILE) {
+    try {
+      $cert_filename = (Get-Item $env:SSL_CERT_FILE).Name
+      $env:SSL_CERT_FILE = Join-Path "Habitat:\hab\cache\ssl" $cert_filename
```

Comment:
Should be `$env:SSL_CERT_FILE = Join-Path $env:HAB_CACHE_SSL_PATH $cert_filename`
Only powershell understands the `Habitat:\` drive but `$env:HAB_CACHE_SSL_PATH` is the cannonical path.